UseCasesofOnlineAdvertising.md

```diff
@@ -3,15 +3,49 @@
 Discussion Paper for the  
 W3C Advertising Business Group
 
-Wendell Baker <[wbaker@verizonmedia.com](mailto:wbaker@verizonmedia.com)\>  
-Initial draft, 2019-03-05
+Contibutors:
+* Wendell Baker <[wbaker@verizonmedia.com](mailto:wbaker@verizonmedia.com)\>  
+* Michael Kleber <[kleber@google.com](mailto:kleber@google.com)\>
+
+Draft, updated 2019-04-17
 
 This document describes event sequences supporting modern online advertising. Many details are elided. There is a significant amount of tutorial literature available which can be used to fill in the details and omissions that are necessarily present in this summary document.
 
-## Ad Call with Single Market
+## Dramatis Personae
+
+The web advertising ecosystem is a collaboration among many parties.  Common designations like "third-party" can often be ambiguous when a single pageload involves interactions with servers run by many organizations with different roles, needs, and capabilities.  So we begin with a list of types of parties and what roles they play.
 
+* **Publisher:**
+  Publishers produce web pages that users visit.  Most of the time, you can think of the publisher as the first party (though aggregation and syndication sites can add a little confusion).
```

Comment:
Is it worth pointing out the user's perspective? I.e. first-party means the user has often taken some action to navigate to that page, would likely say that is where they "are" if asked, and is the URL/origin/brand they see at the top of their browser. These user-side properties are unique to publishers.

Reply:
Feel free to suggest a change to this PR, of course!  But seems to me that when a user is visiting an advertiser site, it has all the properties that you mention as well.

Reply:
Tnxx.  For.  All